### PR TITLE
Spell Cast Sounds (work in progress)

### PR DESCRIPTION
**Pull request by ghost**
The very talented Cuboos is working on some spell sounds and I figured I would make a PR for them since they were having trouble with GitHub.

### CODE REVIEW

**Comment by Jordie0608** [comment]
#Featurefreeze2015

**Comment by Cheridan** [comment]
Sprite updates and map updates have traditionally been exempt because those changes were unlikely to cause bugs and we don't want those content creators to be sitting doing nothing for a whole month. This logic also extends to sound updates.

**Comment by Jordie0608** [comment]
Fair enough

File: code/datums/spell.dm [hunk 1]
```diff
@@ -40,6 +40,9 @@ var/list/spells = typesof(/obj/effect/proc_holder/spell) //needed for the badmin
 	var/level_max = 4 //The max possible level_max is 4
 	var/cooldown_min = 0 //This defines what spell quickened four timeshas as a cooldown. Make sure to set this for every spell
 
+	var/playCastSound = 0 //Whether or not the spell has a sound
```
Comment (RemieRichards) [inline code/datums/spell.dm]: this is needless, just check if the sound has a non null sound file in ```castSound```

**Comment by Incoming5643** [comment]
I gotta say that while this is a nice idea, it kind of limits the ability to add new spells down the road unless we get someone to do it who we know will literally never leave.

File: code/datums/spell.dm (hunk 1 above)
Comment (xxalpha) [inline code/datums/spell.dm]: This holyshit, all these playCastSound = 1 lines, my autism.

**Comment by WJohn** [comment]
They don't HAVE to have a sound.

**Comment by RemieRichards** [comment]
tagged because: 1: WIP 2: Not happy with the implementation, explained above.

**Comment by Xhuis** [comment]
Closing because Cuboos is working on this rather than just providing sounds, honk

**Pull request closed without merging**